### ISSUE TITLE
[Bug]: Accessibility - Dropdown menus like account switcher or theme switcher don't work with screen readers

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [bulwarkmail/webmail#719]
### Description

I'll use the account switcher as an example here, but the same applies to other dropdown menus like the theme switcher on the login page.

Whilst the account switcher dropdown at the top has the correct ARIA roles to be perceived as a dropdown menu by screen readers, clicking it doesn't trigger anything on mobile and drops me in an unusable position, focused on a blank menu, in Windows.

### Steps to Reproduce

For mobile:

1. Enable VoiceOver on iOS
2. Navigate to the account switcher button at the top
3. Double tap it to trigger it
4. Nothing happens. No additional controls become visible anywhere on the page, as near as I could tell.

For Windows:

1. Enable NVDA
2. Navigate to the account switcher button.
3. Click it (space bar/enter)
4. You will be dropped into a menu, but the menu is empty; keybaord navigation doesn't work, and disengaging focus mode doesn't reveal any additional elements in the page where you could change accounts.

### Expected Behavior

Triggering the button should drop the user into a menu where they can select the account they want to switch to. Focus should move to the first menu item, and keyboard navigation with up and down arrow keys should be possible on Windows and other desktop platforms.

### Actual Behavior

Nothing happens, likely because the account switcher only works when mouse events like hover are fired; why exactly it doesn't work on iOS, I'm not sure, but I suspect necessary ARIA dattributes to make this work are missing.

### Bulwark Version

1.7.8

### Stalwart Mail Server Version

0.16.15

### Browser

Chrome / Chromium, Safari, Firefox

### Operating System

iOS

### Relevant Logs or Error Output

```shell

```

**Comment by Almost-Senseless-Coder** [comment]
After further experimentation, I realised that switching accounts is possible if you first click the Toggle menu button, then click on the account selection drop down, then jump to the very bottom of the page; there, the relevantcontrols havebeen inserted into the UI. However, since the sidebar menu is always visible to screen readers (see #721 ) this isn't intuitive. Furthermore, thekeyboard focus should move to the firstavailable account once the account switchermenu has become available.
